Solutions/InsiderRiskManagement/Hunting Queries/InsiderAnomalousAccess.yaml

```diff
@@ -0,0 +1,58 @@
+id: 3bc01058-ce23-45ba-a7e9-409d844b12dc
+name: Insider Risk - Anomalous Access
+description: |
+  'The User Threshold for Anomalous Access is Exceeded'
+requiredDataConnectors:
+  - connectorId: BehaviorAnalytics
+    dataTypes:
+      - BehaviorAnalytics
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - SigninLogs  
+tactics:
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1078
+query: |
+  let UncommonActionVolume = BehaviorAnalytics
+      | extend UncommonActionVolume = tostring(ActivityInsights.UncommonHighVolumeOfActions)
+      | where UncommonActionVolume == "True"
+      | summarize count() by UserPrincipalName
+      | project-rename UncommonActionVolume = count_;
+  let UncommonAction = BehaviorAnalytics
+      | extend UncommonAction = tostring(ActivityInsights.ActionUncommonlyPerformedByUser)
+      | where UncommonAction == "True"
+      | summarize count() by UserPrincipalName
+      | project-rename UncommonAction = count_;
+  let Uncommon = UncommonActionVolume
+      | join(UncommonAction) on UserPrincipalName;
```

Comment:
Please include join kinds with the query, LA will notify you of the recommendation and it is highly recommended.  Example - | join kind = inner